src/right-field/results-table/ResultsTable.jsx

```diff
@@ -43,15 +42,15 @@ function ResultsTable(props) {
  * @param {Integer} size total amount of variables
  * @returns {Object} a configuration object corresponding to a column, following GridJS config. 
  */
-function generateColumn(variable, size) {
+function generateColumn(variable, size, isQuerying) {
```

Comment:
Where is `isQuerying` used in this function? It's also not mentioned in the documentation.

Reply:
The function documentation was outdated, now updated in [commit 1490ba7](https://github.com/SolidLabResearch/generic-data-viewer/pull/23/commits/1490ba7ed068675ef413a3126990dbf53fdb5382)